Ports/AvailablePorts.md

```diff
@@ -6,11 +6,23 @@ This list is also available at [ports.serenityos.net](https://ports.serenityos.n
+| [`RISCVEmu`](RISCVEmu/)                             | A Basic C++ RISC-V Emulator                                     | ad8ad6a                  | https://github.com/IdanHo/RISCVEmu                                             |
+| [`RetroArch`](RetroArch/)                           | RetroArch                                                       | 1.10.3                   | https://www.retroarch.com                                                      |
+| [`SDL2`](SDL2/)                                     | Simple DirectMedia Layer (SDL2)                                 | 2.0.22                   | https://github.com/libsdl-org/SDL                                              |
+| [`SDL2-GNUBoy`](SDL2-GNUBoy/)                       | SDL2 GNUBoy                                                     | 1.2.1                    | https://github.com/AlexOberhofer/SDL2-GNUBoy                                   |
+| [`SDL2_gfx`](SDL2_gfx/)                             | SDL2\_gfx (Graphics primitives add-on for SDL2)                 | 1.0.4                    | https://sourceforge.net/projects/sdl2gfx/                                      |
+| [`SDL2_image`](SDL2_image/)                         | SDL2\_image (Image loading add-on for SDL2)                     | 2.0.5                    | https://www.libsdl.org/projects/SDL_image/                                     |
+| [`SDL2_mixer`](SDL2_mixer/)                         | SDL2\_mixer (audio mixer add-on for SDL2)                       | 2.0.4                    | https://www.libsdl.org/projects/SDL_mixer/                                     |
+| [`SDL2_net`](SDL2_net/)                             | SDL2\_net (network add-on for SDL2)                             | 2.0.1                    | https://www.libsdl.org/projects/SDL_net/                                       |
+| [`SDL2_ttf`](SDL2_ttf/)                             | SDL2\_ttf (TrueType Font add-on for SDL2)                       | 2.0.18                   | https://www.libsdl.org/projects/SDL_ttf/                                       |
+| [`SDLPoP`](SDLPoP/)                                 | Prince of Persia game                                           |                          | https://github.com/NagyD/SDLPoP                                                |
+| [`SDL_sound`](SDL_sound/)                           | SDL_sound (Abstract soundfile decoder)                          |                          | https://github.com/icculus/SDL_sound                                           |
+| [`Super-Mario`](Super-Mario/)                       | Super-Mario Clone                                               |                          | https://github.com/Bennyhwanggggg/Super-Mario-Clone-Cpp                        |
```

Comment:
This is a weird way of ordering the ports. Can we make it truly alphabetically?

Reply:
It is the same way as the GitHub sort and Python by default, alternative sorts are available like `key=str.lower` which place uppercase ones inline.

Reply:
Inline sort would be much better in my opinion.

Reply:
Changed